features/iotserver-feature/org.wso2.carbon.device.mgt.iot.feature/src/main/resources/jaggeryapps/store/extensions/app/store-device-type/themes/store/js/policy-add-new.js

```diff
@@ -0,0 +1,225 @@
+var deviceId, deviceType;
+$('select.select2').select2({
+    placeholder: 'Select..'
+});
+
+$('select.select2[multiple=multiple]').select2({
+    placeholder: 'Select..',
+    tags: true
+});
+
+var stepperRegistry = {},
+    hiddenOperation = '.wr-hidden-operations-content > div',
+    advanceOperation = '.wr-advance-operations';
+
+function initStepper(selector) {
+    $(selector).click(function () {
+        var nextStep = $(this).data("next");
+        var currentStep = $(this).data("current");
+        var isBack = $(this).data("back");
+        if (!isBack) {
+            var action = stepperRegistry[currentStep];
+            if (action) {
+                action(this);
+            }
+        }
+        if (!nextStep) {
+            var direct = $(this).data("direct");
+            window.location.href = direct;
+        }
+        $(".itm-wiz").each(function () {
+            var step = $(this).data("step");
+            if (step == nextStep) {
+                $(this).addClass("itm-wiz-current");
+            } else {
+                $(this).removeClass("itm-wiz-current");
+            }
+        });
+        $(".wr-wizard").html($(".wr-steps").html());
+        $("." + nextStep).removeClass("hidden");
+        $("." + currentStep).addClass("hidden");
+
+    });
+}
+
+function showAdvanceOperation(operation, button) {
+    $(button).addClass('selected');
+    $(button).siblings().removeClass('selected');
+    $(hiddenOperation + '[data-operation="' + operation + '"]').show();
+    $(hiddenOperation + '[data-operation="' + operation + '"]').siblings().hide();
+}
+
+var policy = {};
+var configuredProfiles = [];
+
+function savePolicy() {
+
+    var payload = {
+        policyName: policy.policyName,
+        compliance: policy.selectedAction,
+        ownershipType: policy.selectedOwnership,
+        profile: {
+            profileName: policy.policyName,
+            deviceType: {
+                id: policy.devicetypeId,
+                name: policy.devicetype
+            },
+            policyDefinition: policy.policyDefinition,
+            policyDescription: policy.policyDescription
+        }
+    };
+
+    invokerUtil.post("../../apis/policies/add", payload, function (data, txtStatus, jqxhr) {
+        $(".policy-message").removeClass("hidden");
+        $(".add-policy").addClass("hidden");
+        setTimeout(function () {
+            window.location = "../policies";
+        }, 1500);
+    }, function (err) {
+        console.log(err);
```

Comment:
Remove this console.log(err) from here and associate any appropriate error handling logic here.